kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -499,6 +501,63 @@ object KyuubiConf {
     .checkValue(p => p == 0 || (p > 1024 && p < 65535), "Invalid Port number")
     .createWithDefault(10099)
 
+  val FRONTEND_MYSQL_BIND_HOST: OptionalConfigEntry[String] = buildConf("frontend.mysql.bind.host")
```

Comment:
is it better to create a new file `KyuubiMysqlConf` to define these configs ?

Reply:
Then we also need to extract thrift and rest confs

Reply:
maybe we can do this in a new PR